docs/sections/user_guide/installation.rst

```diff
@@ -1,5 +1,56 @@
+.. include:: ../links.rst
+
 ************
 Installation
 ************
 
-TBD
+
+-  If you are a Developer, skip the User Installation and continue on to the `Developer Setup`_ located in the Contributor Guide.
```

Comment:
Working on this Developer Setup relative link, I tried `Developer Setup <developer_setup.html>` and didn't have luck. Still working on this.

Reply:
For example, if right now I visit https://unified-workflow--355.org.readthedocs.build/en/355/sections/user_guide/installation.html and click the "Developer Setup" link in the bullet point at the very top, I get a 404. But if I right-click on that link and click _Inspect_ to open it in Chrome's Developer Tools, right-click on `developer_setup.html`, click _Edit as HTML_, and change the URL to `../contributor_guide/developer_setup.html`, the link then works. (Changes made in Chrome's Developer Tools are ephemeral and will be lost as soon as the page is reloaded, but this can be a handy want to test changes on a live page, all in your browser.)

The URL hierarchy mirrors the way the HTML and other asset files are laid out on disk, so you can look at the `docs/build` directory in your git clone after you do `make html` to see how the files will be laid out on disk when RTD deploys them.

I thought there might also be an RST way to create a relative link, but the HTML way should work fine.